.github/workflows/pants.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
     - name: Bootstrap Pants
       run: pants --version
     - name: Check Pants config files
-      run: pants tailor --check update-build-files --check
+      run: pants tailor --check update-build-files --check '::'
```

Comment:
Otherwise

```
14:40:59.29 [WARN] No arguments specified with `pants tailor`, so the goal will do nothing.

Instead, you should provide arguments like this:

  * `pants tailor ::` to run on everything
  * `pants tailor dir::` to run on `dir` and subdirs
  * `pants tailor dir` to run on `dir`
  * `pants --changed-since=HEAD tailor` to only run on changed and new files
14:40:59.29 [WARN] No arguments specified with `pants update-build-files`, so the goal will do nothing.

Instead, you should provide arguments like this:

  * `pants update-build-files ::` to run on everything
  * `pants update-build-files dir::` to run on `dir` and subdirs
  * `pants update-build-files dir` to run on `dir`
  * `pants update-build-files dir/BUILD` to run on that single BUILD file
  * `pants --changed-since=HEAD update-build-files` to run only on changed BUILD files
```